src/components/Navbar.vue

```diff
@@ -28,16 +31,47 @@
 
 <script>
 export default {
-  name: 'Navbar'
+  name: 'Navbar',
+  data() {
+    return {
+      fillNavbar: false
+    }
+  },
+  mounted () {
+    window.addEventListener('scroll', this.onScroll)
+  },
+  beforeDestroy () {
+    window.removeEventListener('scroll', this.onScroll)
+  },
+  methods: {
+    onScroll () {
+      // Get the current scroll position
+      const currentScrollPosition = window.pageYOffset || document.documentElement.scrollTop
+      // Because of momentum scrolling on mobiles, we shouldn't continue if it is less than zero
+      if (currentScrollPosition < 0) {
+        return
```

Comment:
You should not write `return` without an argument. This is very dangerous. Let's add an `undefined` after it at least.

Reply:
I'll outright delete this conditional. Investigating a little more I see that It's not really needed in this case.